test/test-applications/integrations/TestApplication.StackExchangeRedis/TestApplication.StackExchangeRedis.csproj

```diff
@@ -1,11 +1,14 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="StackExchange.Redis" Version="2.1.58" />
+    <!-- Workaround! System.Runtime.CompilerServices.Unsafe v. 6.0.0 is minimal version supportef by auto instrumentation.
```

Comment:
Starting 2.2.3 version of `StackExchange.Redis`, it does not have reference to `System.Runtime.CompilerServices.Unsafe`. Should we upgrade the package here, 2.2.3 is the version released after 2.1.58?

Reply:
I would say, that we should keep older version. Based on our testing strategy we probably should use even older version, but it is good version as it can instrument both normal, and sentinel connections. See https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/82a5ad9d3ce6cb0e3d301e5b31d624b5ddb4b05b/test/test-applications/integrations/TestApplication.StackExchangeRedis/Program.cs#L61

I will extend comment to include this information.